backend/convert_from_protobuf.go

```diff
@@ -248,10 +248,14 @@ func (f ConvertFromProtobuf) CollectMetricsResponse(protoResp *pluginv2.CollectM
 
 // SubscribeStreamRequest ...
 func (f ConvertFromProtobuf) SubscribeStreamRequest(protoReq *pluginv2.SubscribeStreamRequest) *SubscribeStreamRequest {
+	if protoReq.Headers == nil {
+		protoReq.Headers = map[string]string{}
+	}
```

Comment:
curious, can't protoReq.Headers be nil?

Reply:
It's just the same convention we have to always set to empty map for the other pre-existing header maps 🤷